--- threads-2.21/threads.xs.orig	2018-01-23 02:10:14.000000000 +0100
+++ threads-2.21/threads.xs	2024-07-01 21:07:39.076104230 +0200
@@ -770,7 +770,8 @@
           int fd = PerlIO_fileno(Perl_error_log);
           if (fd >= 0) {
             /* If there's no error_log, we cannot scream about it missing. */
-            PERL_UNUSED_RESULT(PerlLIO_write(fd, PL_no_mem, strlen(PL_no_mem)));
+            static const char oomp[] = "Out of memory in perl:threads:ithread_create\n";
+            PERL_UNUSED_RESULT(PerlLIO_write(fd, oomp, sizeof oomp - 1));
           }
         }
         my_exit(1);
